frb/data/Galaxies/121102/FRB121102_host.json

```diff
@@ -47,12 +53,12 @@
         "[OIII] 5007_err": 8.379935678693262e-18
     },
     "offsets": {
-        "ang_avg": 0.1245685453761799,
-        "ang_avg_err": 0.0027529576372375447,
-        "ang_best": 0.12450060896549818,
-        "ang_best_err": 0.002753795763727014,
-        "physical": 0.41179423877492893,
-        "physical_err": 0.009108366936420729
+        "ang_avg": 0.22738211697817798,
+        "ang_avg_err": 0.015495714927742417,
+        "ang_best": 0.2268624805850244,
+        "ang_best_err": 0.015504425274835199,
+        "physical": 0.7503630968181939,
```

Comment:
Looks too high, Bassa+17 estimate physical offset = 0.18 kpc (and it's even higher than our own initial analysis
)

Reply:
That offset is from the peak of the SF not the center of the galaxy.